internal/operators/cnv/cnv_operator_test.go

```diff
@@ -352,29 +349,37 @@ var _ = Describe("CNV operator", func() {
 	})
 
 	Context("cluster requirements", func() {
-		It("only x86_64 is supported for CNV operator", func() {
-			cluster := common.Cluster{}
+		cluster := common.Cluster{}
+		table.DescribeTable("Validate Cluster", func(ocpVersion []string, cpuArch string, expectedApiStatus api.ValidationStatus) {
+			cluster.CPUArchitecture = cpuArch
+			for _, v := range ocpVersion {
+				cluster.OpenshiftVersion = v
+				validation, err := operator.ValidateCluster(context.TODO(), &cluster)
+				Expect(err).ToNot(HaveOccurred())
 
-			cluster.CPUArchitecture = common.DefaultCPUArchitecture
-			validation, err := operator.ValidateCluster(context.TODO(), &cluster)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(validation.Status).To(Equal(api.Success))
+				Expect(validation.Status).To(Equal(expectedApiStatus),
 
-			cluster.CPUArchitecture = "arm64"
-			validation, err = operator.ValidateCluster(context.TODO(), &cluster)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(validation.Status).To(Equal(api.Failure))
-			Expect(validation.Reasons).To(ContainElements(
-				"OpenShift Virtualization is supported only for x86_64 CPU architecture."))
-		})
-		It("multi-arch is supported for CNV operator", func() {
-			cluster := common.Cluster{}
+					fmt.Sprintf("Validate OCP version %s, in %s validatation should be %s", cluster.OpenshiftVersion, cpuArch, expectedApiStatus))
 
-			cluster.CPUArchitecture = common.MultiCPUArchitecture
-			validation, err := operator.ValidateCluster(context.TODO(), &cluster)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(validation.Status).To(Equal(api.Success))
-		})
+			}
+
+		},
+			table.Entry("on X86 Support all Versions",
```

Comment:
`common.MultiCPUArchitecture` is missing.

Reply:
there isn't any validation for `common.MultiCPUArchitecture` so it will pass on all versions.
maybe we should think about adding it.